index.html

```diff
@@ -2,39 +2,59 @@
 <html lang="en">
 
 <head>
-   <!-- Global site tag (gtag.js) - Google Analytics -->
-<script async src="https://www.googletagmanager.com/gtag/js?id=G-HDWG0CLCER"></script>
-<script>
-  window.dataLayer = window.dataLayer || [];
-  function gtag(){dataLayer.push(arguments);}
-  gtag('js', new Date());
-
-  gtag('config', 'G-HDWG0CLCER');
-</script>
+
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <link rel="stylesheet" href="desktop.css">
-    <script src="https://kit.fontawesome.com/bf723a76a7.js" crossorigin="anonymous"></script>
     <title>Document</title>
+
+    <script src="https://kit.fontawesome.com/bf723a76a7.js" crossorigin="anonymous"></script>
+    <script defer src="script.js"></script>
+
 </head>
 
 <body>
-    <header>
-        <div class="header_elements name">
-            <h3>ikpro</h3>
-        </div>
+    <header id="hd">
 
-        <div class="hcontent">
-            <a href=# class="header_elements desktop hportfolio">Portfolio</a>
-            <a href=# class="header_elements desktop habout">About</a>
-            <a href=# class="header_elements desktop hcontact">Contact</a>
-            <a href=# class="header_elements desktop hletters"><i class="fa-solid fa-envelope"></i></a>
-            <div class="header_elements Icons"><a href=#><i class="fa-solid fa-bars"></i></a></div>
-        </div>
+
+        <ul class="hcontent" id = "eposition">
+            <li>
+                <div class="navstyle">
+                    <h3 id="john">John Doe</h3>
+                    <a href=# id="toggle"><i class="fa-solid fa-bars"></i></a>
+                    <a href=# class="tattempt"><i class="fa-solid fa-bars"></i></a>
+                </div>
+            </li>
+            <li>
+                <div class="navstyle headvisible">
+                    <h3>Portfolio</h3>
+                    <a href=#><i class="fa-solid fa-angle-right"></i></a>
+                </div>
+            </li>
+            <li>
+                <div class="navstyle headvisible">
+                    <h3>Contact</h3>
+                    <a href=#><i class="fa-solid fa-angle-right"></i></a>
+                </div>
+            </li>
+            <li>
+                <div class="navstyle headvisible">
+                    <h3>About</h3>
+                    <a href=#><i class="fa-solid fa-angle-right"></i></a>
+                </div>
+            </li>
```

Comment:
- [ ]  Good job the design, But you are supposed to stick to the [figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) design as much as possible. Your design is a little different from the template. Please use correct font size, weight, and correct color for your buttons and make sure it matches the design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/157854145-6ccb6ea5-0fd6-4223-834d-706c117cb5a1.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/157854264-a8b5706a-5599-43f5-801a-50acc161c3df.png)